docker-compose.yml

```diff
@@ -1,81 +1,71 @@
-version: '3'
+version: "3"
```

Comment:
@lacexd please revert changes in this file, so I can merge your PR

Reply:
@eldadfux reverted it, sorry